proposals/0002-Extracted-Command-Line-Tools.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Extract CLI from Core
+author:
+- Mike Grabowski
+date: 2018-08-10
+---
+
+# RFC0000: Extract CLI from Core
+
+## Summary
+
+Extract the current CLI implementation from React Native Core and move it into a third party package.
+
+## Basic example
```

Comment:
Should we just call this section "Background", even in the template ? Looks like almost all proposals seem to need a background section, talking about the state of the world.